### PR TITLE
Improve performance of attributes setting 

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Accessing a string's character at index allocates a new, single character string.
A better (faster) check is to use `charCodeAt` that doesn't trigger allocation.

This simple change speeds up the element_text_create benchmark by ~7%.

### CODE REVIEW

**Review by mhevery — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
- [VE Presubmit](http://test/OCL:272895484:BASE:272895517:1570207620904:f9f09dd)
- [Ivy Presubmit](http://test/OCL:272895484:BASE:272895676:1570207676890:abbff36c)

**Comment by AndrewKushnir** [comment]
FYI, VE and Ivy presubmits are successful.

File: packages/core/src/render3/util/attrs_utils.ts [hunk 1]
```diff
@@ -100,8 +100,6 @@ export function isNameOnlyAttributeMarker(marker: string | AttributeMarker | Css
       marker === AttributeMarker.I18n;
 }
 
-export const ANIMATION_PROP_PREFIX = '@';
-
 export function isAnimationProp(name: string): boolean {
-  return name[0] === ANIMATION_PROP_PREFIX;
+  return name.charCodeAt(0) === 64;  // @
```
Comment (kara) [inline packages/core/src/render3/util/attrs_utils.ts]: Can we add a small comment that explains the perf difference so no one changes it later?

**Review by kara — APPROVED** [review]
LGTM, one nit

File: packages/core/src/render3/util/attrs_utils.ts (hunk 1 above)
Comment (pkozlowski-opensource) [reply packages/core/src/render3/util/attrs_utils.ts]: Done

**Comment by alxhub** [comment]
[Presubmit](http://test/OCL:273321440:BASE:273321479:1570469079379:83e98ddb)

**Pull request closed without merging**